Automattic/jetpack
# AutomatticBylineLogo | JetpackLogo: Changing `title` tag to `desc`

Using `title` inside `svg` go against [W3C](https://www.w3.org/TR/html401/struct/global.html#h-7.4.2).

#### Changes proposed in this Pull Request:

* Changing `title` for `desc`.

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Start `Storybook`
* Go to `AutomatticBylineLogo` story.
* You should see `desc` tag instead `title`.
